### ISSUE TITLE
Window and menu interactions can lead to rapid high contrast flickering

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#125]
Make your window slightly shorter than the height of the options menu and load few enough accounts that the browser doesn't generate a scrollbar.

Mouse over options entering on the left edge. The options menu opens and generates a scrollbar on the right edge of the browser which pushes the menu left 20px.

Exit options on the right heading towards setup button.

Once your mouse enters the setup button div it will close the options menu and remove the scrollbar. This shifts the menu right 20px and places the options button underneath your mouse again. This creates an open/close loop which causes extreme, potentially full-screen black/white flashing.

Basically, this is an epilepsy risk that needs to be fixed.

**Comment by jakcodex** [comment]
The scrollbar is now set to always render.
